app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/home/HomeScreen.kt

```diff
@@ -10,7 +10,6 @@ import com.ramcosta.composedestinations.annotation.Destination
 @Composable
 fun HomeScreen() {
     val activity = LocalContext.current as? Activity
-
     BackHandler {
         activity?.finish()
```

Comment:
We could inline this val 🤔 
```suggestion
       (LocalContext.current as? Activity)?.finish()
```

Reply:
I am going to update this in the `Home screen integration PR`